### ISSUE TITLE
Display of truncated tasks broken since 1.3.1

### ISSUE THREAD

**Issue by JochenDiekenbrock** [angular-gantt/angular-gantt#729]
I recently upgraded our app from angular-gantt to 1.3.1. The upgrade breaks how truncated tasks are displayed in our app. 
Previously (with 1.3.0), the div with class gantt-task had something like style="left: 0px; width: 590px;"
With 1.3.1, left/right is now going into negative values, for example style="left: -2360px; width: 2832px;"

This causes that the task content is not displayed as expected (it's shifted out of sight) and that the truncation markers are not visible.

Looking through the latest commits, the fix for issue 262 (Progress width is not valid for truncated tasks) could be the reason for the changed behaviour.

Is there a way to fall back to the previous behaviour? Any other ideas how to resolve the above?

**Comment by Toilal** [comment]
You're right. Also, the label of the task may be hidden because it's placed in center of this div.

**Comment by Toilal** [comment]
But the truncated icon (< or >) seems to be displayed properly on master ([unstable demo](http://rawgit.com/angular-gantt/angular-gantt/master/demo/dist/index.html).

**Comment by Daemon-Knoppix** [comment]
Hi @Toilal, is there any ETA for this issue? Or any workaround to achieve this? I am facing same issue for showing a task longer than a month and name and progress is not visible due to tasks full width.
